### PR TITLE
qt4:Improve a diolog of contacts invitation

### PR DESCRIPTION
**Pull request by jamesaxl**
:)

### CODE REVIEW

File: emesene/gui/qt4ui/Dialog.py [hunk 1]
```diff
@@ -520,11 +521,6 @@ def yes_no(cls, message, response_cb, *args):
         response_cb(response, *args)
 
     @classmethod
-    def contact_information_dialog(cls, session, account):
```
Comment (c10ud) [inline emesene/gui/qt4ui/Dialog.py:523]: why did you remove this?

**Comment by c10ud** [comment]
If i'm right you're creating widgets for every account..we don't always add tons of new accounts but this is inefficient imho.

 you can put contact infos in a page and reuse the same buttons, just check which is the active page and act accordingly

if i'm mistaken please correct me

**Comment by c10ud** [comment]
however this was just a thought on the code, if you restore the classmethod you removed unrelated to this pull request i think i can pull this

**Comment by jamesaxl** [comment]
yeah you are right, i just wanted make it like gtkui, ok i'm going to restore classmethod
